ytypes/leaf.go

```diff
@@ -73,7 +79,7 @@ func validateLeaf(inSchema *yang.Entry, value interface{}) (errors []error) {
 		return nil
 		// TODO(mostrowski): restore when representation is decided.
 		//return util.AppendErr(errors, validateBitset(schema, rv))
-	case yang.Ybool:
+	case yang.Ybool, yang.Yempty:
```

Comment:
Since handling is changing, it would be better to create a new function validateEmpty.

Reply:
Done